src/main/resources/site/content-types/combinedGraph/combinedGraph.xml

```diff
@@ -60,6 +60,10 @@
                 <default>0</default>
                 <help-text>Her velger du hvor mange desimalplasser du vil vise i y-akse</help-text>
             </input> 
+            <input name="yAxisOffset" type="TextLine">
+              <label>Plassering av y-aksetittel</label>
```

Comment:
Lurte bare litt på om vi skulle brukt "justering av y-aksetittel" i stedet for "plassering"? Evt. "horisontal justering av y-aksetittel"?